rpcd/playbooks/roles/rpc_support/tasks/main.yml

```diff
@@ -41,11 +41,11 @@
 
 - include: support_key_pair.yml
   when: >
-    inventory_hostname == groups['utility'][0]
+    groups['nova_api_os_compute']|length > 0 and inventory_hostname == groups['utility'][0]
```

Comment:
Stylistically, should this be a length check, or check if index 0 is defined?

`groups['nova_api_os_compute'][0] is defined`